src/main/java/com/yugi/application/dao/mapper.java

```diff
@@ -17,4 +17,7 @@ public interface Mapper {
 
     @Insert("INSERT INTO users(userName, passWord) VALUES(#{userName}, #{passWord})")
     void addUser(User user);
+
+    @Insert("DELETE FROM users where id = ${id})")
+    void removeUser(User user);
```

Comment:
**Suggestion:** Correct the annotation for the removeUser method from @Insert to @Delete.
```suggestion
    @Delete("DELETE FROM users where id = #{id}")
    void removeUser(@Param("id") long id);
```